src/app/pages/adoption-form/book-info/book-info.js

```diff
@@ -0,0 +1,74 @@
+import BaseView from '~/helpers/backbone/view';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './book-info.hbs';
+
+function requireOne(container) {
+    let checkboxes = container.querySelectorAll('input[type=checkbox]'),
+        checkCount = 0,
+        setClass = function () {
```

Comment:
Just a stylistic preference, but I think this might look cleaner as a function expression instead of a function declaration (it would also remove a level of indentation).